Makefile

```diff
@@ -1,9 +1,27 @@
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DELETE_ON_ERROR:
+
+# add easyjson source file targets here
+easyjson_sources := basm/dto.go x/xbasm/parse.go
+easyjson_generated := $(easyjson_sources:.go=_easyjson.go)
+
+# Rule to generate *_easyjson.go files
+%_easyjson.go: %.go
+	@easyjson $<
+
+# Generate all *_easyjson.go files
+generate: $(easyjson_generated)
+
+.PHONY: lint
 lint:
```

Comment:
the styleguide suggested adding .PHONY to all targets that do not represent a generated filename (i.e. any commands that are intended to be task runners rather than targets that result in a file)